code/obj/item/clothing/gimmick.dm

```diff
@@ -89,7 +89,6 @@
 	desc = "Twoooo!"
 	icon_state = "owl"
 	item_state = "owl"
-	compatible_species = list("human", "monkey")
```

Comment:
Monkeys already have `human_compatible` and `uses_human_clothes` as `TRUE`, so they actually don't need to be specifically added to any `compatible_species` lists (because they're covered by humans).